lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.ts

```diff
@@ -89,7 +91,7 @@ export class SearchChipAutocompleteInputComponent implements OnInit, OnDestroy,
     formCtrl = new FormControl('');
     filteredOptions: AutocompleteOption[] = [];
     selectedOptions: AutocompleteOption[] = [];
-    private onDestroy$ = new Subject<void>();
+    private readonly destroyRef = inject(DestroyRef);
```

Comment:
can we make 3 separated groups? Public fields, private fields, private readonly fields and each group separated by empty line?